common/test/acceptance/tests/lms/test_lms.py

```diff
@@ -596,6 +608,48 @@ def test_edit_course_wiki(self):
         actual_content = unicode(self.course_wiki_page.q(css='.wiki-article p').text[0])
         self.assertEqual(content, actual_content)
 
+    @attr('a11y')
+    def test_view_a11y(self):
+        """
+        Verify the basic accessibility of the wiki page as initially displayed.
+        """
+        self._check_for_accessibility_errors(self.course_wiki_page)
+
+    @attr('a11y')
+    def test_edit_a11y(self):
+        """
+        Verify the basic accessibility of edit wiki page.
+        """
+        self._open_editor()
+        self._check_for_accessibility_errors(self.course_wiki_edit_page)
+
+    @attr('a11y')
+    def test_changes_a11y(self):
+        """
+        Verify the basic accessibility of changes wiki page.
+        """
+        self.course_wiki_page.show_changes()
+        changes_page = CourseWikiChangesPage(self.browser, self.course_id, self.course_info)
+        changes_page.wait_for_page()
+        self._check_for_accessibility_errors(changes_page)
+
+    @attr('a11y')
+    def test_children_a11y(self):
+        """
+        Verify the basic accessibility of changes wiki page.
+        """
+        self.course_wiki_page.show_children()
+        children_page = CourseWikiChildrenPage(self.browser, self.course_id, self.course_info)
+        children_page.wait_for_page()
+        custom_rules = {
+            'ignore': [
+                'section',  # AC-714, bug in the custom rule
+                'label',  # TNL-6440
```

Comment:
'label' and 'data-table' will require changes in our forked version of django-wiki (which was last edited over 2 years ago). https://github.com/edx/django-wiki

Given the additional complexity, I thought it best to not roll those changes into this PR.